src/Stratis.Features.FederatedPeg/TargetChain/Withdrawal.cs

```diff
@@ -42,7 +42,7 @@ public override string ToString()
         public string GetInfo()
         {
             return string.Format("Tran#={0} Dep#={1} Amount={2,12} Addr={3} BlkNum={4,8} BlkHash={5}",
-                this.Id.ToString().Substring(0, 6),
+                this.Id?.ToString().Substring(0, 6),
```

Comment:
How is this possible?

Reply:
In this case it's just because the class is being reused to help transportation to ComponentStats.